plugins/woocommerce/src/Internal/Orders/PaymentInfo.php

```diff
@@ -112,24 +112,22 @@ private static function get_wcpay_card_info( WC_Abstract_Order $order ): array {
 				return array();
 			}
 
-			$intent_id = $order->get_meta( '_intent_id' );
-			if ( ! $intent_id ) {
+			$payment_method_id = $order->get_meta( '_payment_method_id' );
+			if ( ! $payment_method_id ) {
 				return array();
 			}
 
 			try {
 				$payment_details = \WC_Payments::get_payments_api_client()
-					->get_intent( $intent_id )
-					->get_charge()
-					->get_payment_method_details();
+					->get_payment_method( $payment_method_id );
```

Comment:
Does this return the same data, in the same structure and format, as `get_payment_method_details`? It seems to in my testing, but just wondering if there could be an edge case where it would be different.

Reply:
Yes, `get_payment_method` returns the same format of the payment method details, as `get_payment_method_details`. The format comes from Stripe, and AFAICT, we do not modify it any point.